pyproject.toml

```diff
@@ -45,26 +45,26 @@ classifiers = [
 [tool.poetry.dependencies]
 python = "^3.9.0"
 cryptography = "^37.0.0"
-flask = "^2.1.0"
-flask-wtf = "^1.0.0"
+flask = "^2.0.1"
+flask-wtf = "^0.14.3"
```

Comment:
We do want to support 1.0.0, so it should be something like `">=0.14.3, <2.0.0"`